src/diffpy/labpdfproc/labpdfprocapp.py

```diff
@@ -63,11 +63,10 @@ def get_args():
         "--user-metadata",
         metavar="KEY=VALUE",
         nargs="+",
-        help="Specify (multiple) key-value pairs to be loaded into metadata using the format key=value. "
-        "Please separate each pair with whitespace, and make sure no whitespaces before or after the = sign. "
-        "Avoid using = in keys. If multiple = signs are present, only the first one separates the key and value. "
-        "If you repeat key names, it will replace previous values. "
-        "If a key or value contains whitespace, enclose it in quotes. "
+        help="Specify key-value pairs to be loaded into metadata using the format key=value. "
+        "Separate pairs with whitespace, and ensure no whitespaces before or after the = sign. "
+        "Avoid using = in keys. If multiple = signs are present, only the first separates the key and value. "
+        "Please do not repeat key names. If a key or value contains whitespace, enclose it in quotes. "
```

Comment:
I think I would remove "please do not repeat key  names" from the help.  This is probably tmi.  I think as a user I eould only do this by mistake and the if I receive an error message saying "dumbass....you gave the same key twice" I would oh, I am a dumbass.    If we make the help too long people won't read and they won't see other, more important this.

Also, along the same lines, I think examples are better than text, so reproduce here some of your test examples of valid, well formed strings.  e.g., say, "for example, ....... are all valid key=value items"